sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/QueryBuilderSamples.cs

```diff
@@ -73,6 +73,54 @@ public static void Main()
                 .Where()
                 .IsOfModel("dtmi:example:room;1", true)
                 .Build();
+            
+            // SELECT * FROM DIGITALTWINS WHERE Temperature = 50 AND IS_OF_MODEL("dtmi..", exact)
+            AdtQueryBuilder logicalOps_SingleAnd = new AdtQueryBuilder()
+                .Select("*")
+                .From(AdtCollection.DigitalTwins)
+                .Where()
+                .Compare("Temperature", QueryComparisonOperator.Equal, 50)
+                .And()
+                .IsOfModel("dtmi:example:room;1", true)
+                .Build();
+
+            // SELECT * FROM DIGITALTWINS WHERE Temperature = 50 OR IS_OF_MODEL("dtmi..", exact)
+            AdtQueryBuilder logicalOps_SingleOr = new AdtQueryBuilder()
+                .Select("*")
+                .From(AdtCollection.DigitalTwins)
+                .Where()
+                .Compare("Temperature", QueryComparisonOperator.Equal, 50)
+                .Or()
+                .IsOfModel("dtmi:example:room;1", true)
+                .Build();
+
+            // SELECT * FROM DIGITALTWINS WHERE Temperature = 50 OR IS_OF_MODEL("dtmi..", exact) OR IS_NUMBER(Temperature)
+            AdtQueryBuilder logicalOps_MultipleOr = new AdtQueryBuilder()
+                .Select("*")
+                .From(AdtCollection.DigitalTwins)
+                .Where()
+                .Compare("Temperature", QueryComparisonOperator.Equal, 50)
+                .Or()
+                .IsOfModel("dtmi:example:room;1", true)
+                .Or()
+                .IsOfType("Temperature", AdtDataType.AdtNumber)
+                .Build();
+
+            // SELECT * FROM DIGITALTWINS WHERE (IS_NUMBER(Humidity) OR IS_PRIMATIVE(Humidity)) OR (IS_NUMBER(Temperature) OR IS_PRIMATIVE(Temperature))
```

Comment:
it might be useful to add this comment in a multi-line statement that helps the user read it better. maybe use multi-line style comments

Reply:
mostly important on conditions that have nested logic baked in.